events/api.py

```diff
@@ -1306,14 +1315,36 @@ def _filter_event_queryset(queryset, params, srs=None):
         dt = parse_time(val, is_start=False)
         queryset = queryset.filter(Q(last_modified_time__gte=dt))
 
-    val = params.get('start', None)
-    if val:
-        dt = parse_time(val, is_start=True)
+    if params.get('format') == 'docx':
+        if not params.get('location'):
+            raise ValidationError(_('Must specify a location.'))
```

Comment:
This validation error could mention that this is a docx renderer only limitation.